forecasting/src/autogluon/forecasting/predictor.py

```diff
@@ -0,0 +1,420 @@
+import logging
+import pprint
+import time
+from typing import Optional, Type, Any, Union, Dict, Tuple, List
+
+import pandas as pd
+
+from autogluon.common.utils.log_utils import set_logger_verbosity
+from autogluon.common.utils.utils import setup_outputdir
+from autogluon.core.scheduler.scheduler_factory import scheduler_factory
+from autogluon.core.utils.decorators import apply_presets
+from autogluon.core.utils.loaders import load_pkl
+from autogluon.core.utils.savers import save_pkl
+
+from .configs import FORECASTING_PRESETS_CONFIGS
+from .dataset import TimeSeriesDataFrame
+from .learner import AbstractLearner, ForecastingLearner
+from .trainer import AbstractForecastingTrainer
+
+logger = logging.getLogger()
```

Comment:
`logger = logging.getLogger(__name__)  # return autogluon root logger`

This avoids altering the root logger. Note that this should be done for every logger in every file.